appveyor.yml

```diff
@@ -63,10 +55,10 @@ environment:
 #    secure: UgABryPYZg823OcD1XRosA==
 
   matrix:
-    # Win64-gcc
-    - NC_CMAKE_GENERATOR: "MinGW Makefiles"
-      external_static_subdir: "windows64-gcc"
-      wheel_name_suffix: "win_amd64"
+#    # Win64-gcc
+#    - NC_CMAKE_GENERATOR: "MinGW Makefiles"
```

Comment:
I think we can drop MingGW if MSVC works

Reply:
> I think we can drop MingGW if MSVC works
I started to modify appveyor.yml to replace MingGW with MSVC as you can see but did not complete it.